load-agent/agent.ts

```diff
@@ -149,12 +179,22 @@ const initializeAgent = async (withMediation, port, agentConfig = null) => {
     dependencies: agentDependencies,
     modules: modules
   })
+  
+    const apps = express()
+    const wsTransport = new WsOutboundTransport()
+    const httpTransport = new HttpOutboundTransport()
+    const httpInbound = new HttpInboundTransport({
+      port:4002,
+      app:apps,
```

Comment:
I don't think app:apps, and path are needed here.

Reply:
Fixed